src/k8s_client/k8s_client.go

```diff
@@ -584,3 +585,9 @@ func (c *k8sClient) GetCSVFromSubscription(namespace string, name string) (strin
 	}
 	return result.Status.CurrentCSV, nil
 }
+
+func (c *k8sClient) PatchNodeLabels(node *v1.Node, node_labels string) error {
+	data := []byte(`{"metadata": {"labels": ` + node_labels + `}}`)
```

Comment:
To avoid weird injection / string escaping bugs, please assemble a Metadata struct and serialize it properly rather than using naive string concatenation

Reply:
I am not exactly clear on this. It would be helpful if you can point to an example in the code itself that someone has implemented. Thanks.

Reply:
Oh I now see that node_labels is already a JSON blob, so nevermind. Not too important